rust/stub/src/pe_section.rs

```diff
@@ -13,7 +14,13 @@ pub fn pe_section<'a>(file_data: &'a [u8], section_name: &str) -> Option<&'a [u8
     pe_binary
         .sections
         .iter()
-        .find(|s| s.name().unwrap() == section_name)
+        .find(|s| match s.name() {
+            Ok(n) => n,
+            Err(e) => {
+                error!("No section name found: {}", e);
```

Comment:
Doesn't look like we need to die here. The error message is also misleading, because we only hit this when a section doesn't have a name? Why not warn! ?